WebIDL/ecmascript-binding/constructors-support.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<script>
+window.cons = function() {};
+cons.prototype = 8;
+
+window.MyDOMParser = class extends DOMParser {}
+window.ForeignDOMParser = class extends parent.DOMParser {}
```

Comment:
Clearer variable names might be "DOMParserSubclass" and "ForeignDOMParserSubclass"